### PR TITLE
LTD-3431-copy-amends

### PR DESCRIPTION
**Pull request by depsiatwal**
This jira had multiple issues associated with the flow for change licensing unit case officer followed from the case summary page. 

Changes included copy differences with new Django version and old lite forms version including error messages. 
Some of the copy changes would not have been easy in lite forms. In addition there was also a difference between the flow for this and the flow for assigning case advisor. i.e remove functionality. 

Hence the best option is to add a new remove form for licensing unit case officer and plug the new Django form into the change link on the summary page,. 

https://uktrade.atlassian.net/browse/LTD-3417

### CODE REVIEW

File: caseworker/templates/case/slices/summary.html [hunk 1]
```diff
@@ -188,7 +194,7 @@
 						<li>
                             <div class="app-case__summary-list__value__fullwidth">
 							    <a href="{% url 'users:user' user.id %}?return_to={{ CURRENT_PATH|urlencode }}&return_to_text={{ case.reference_code }}" class="govuk-link govuk-link--no-visited-state">{{ user|username }}</a>
-                                <a href="{% url 'cases:remove-case-assignment' queue.id case.id %}?assignment_id={{user.assignment_id}}" class="govuk-link govuk-link--no-visited-state app-case__summary-list__value__item-action">Remove</a>
```
Comment (currycoder) [inline caseworker/templates/case/slices/summary.html:191]: Doesn't this break the case assignment removal buttons?  Why change this?

**Review by currycoder — COMMENTED** [review]
Seems like unit tests are failing - any ideas what's up?

File: caseworker/templates/case/slices/summary.html (hunk 1 above)
Comment (depsiatwal) [reply caseworker/templates/case/slices/summary.html]: your correct changed by accident reverted back

File: unit_tests/caseworker/cases/views/test_case_assignments.py [hunk 2]
```diff
@@ -119,3 +131,102 @@ def test_case_assignments_remove_user_POST_error(
...
+
+def test_case_remove_case_officer_no_name_GET(
+    authorized_client, data_queue, data_standard_case, mock_standard_case_with_case_officer_no_name, mock_queue
+):
+    case = data_standard_case
+    url = reverse("cases:remove-case-officer", kwargs={"queue_pk": data_queue["id"], "pk": case["case"]["id"]})
+    response = authorized_client.get(url)
+    assert response.status_code == 200
+    context = response.context
+    assert context["case_officer_name"] == "example@example.net"
```
Comment (currycoder) [inline unit_tests/caseworker/cases/views/test_case_assignments.py:159]: Maybe these could be combined with some parameters e.g. `first_name, last_name, email, expected_case_officer_identifier`

Comment (depsiatwal) [reply unit_tests/caseworker/cases/views/test_case_assignments.py:159]: yeah was also considering this so good point

**Review by Harriethw — APPROVED** [review]

**Pull request merged**